README.md

```diff
@@ -10,19 +9,22 @@ Using these abstractions, Pekko also provides libraries for persistence, streams
 
 Pekko is a fork of [Akka](https://github.com/akka/akka) 2.6.x, prior to the Akka project's adoption of the Business Source License.
 
-Reference Documentation
------------------------
+## Reference Documentation
 
 See https://pekko.apache.org for the documentation including the API docs. The docs for all the Apache Pekko modules can be found there.
 
-Building from Source
---------------------
+## Building from Source
 
+### Prerequisites
 - Make sure you have installed a Java Development Kit (JDK) version 11 or later.
 - Make sure you have [sbt](https://www.scala-sbt.org/) installed and using this JDK.
 - Make sure you also have Java 8 installed (JDK or JRE).
-- Use git to clone the [repo](https://github.com/apache/incubator-pekko) or download a source release from https://pekko.apache.org
-- Open a command window and change directory to the directory where you installed the source
+- [Graphvis](https://graphviz.gitlab.io/download/) is needed for the scaladoc generation build task, which is part of the release.
+
+### Running the Build
+- Open a command window and change directory to your preferred base directory
+- Use git to clone the [repo](https://github.com/apache/incubator-pekko) or download a source release from https://pekko.apache.org (and unzip or untar it, as appropriate)
```

Comment:
(offtopic: I wonder if we need to do anything special to make sure the version number is determined correctly when building from a zip/tar)

Reply:
interesting point - let me experiment with that

Reply:
@raboof I ran a build in a dir with the source but that was not under git control. The jars that were built came out with names like:

pekko-remote_2.13-HEAD+20230524-1416.jar

Reply:
HEAD and the date and time